samples/sql/manage_server_key.py

```diff
@@ -179,12 +180,23 @@ def main():
         SERVER_KEY,
         {
             # TODO: init resource body
-            "server_key_type": "AzureKeyVault",
+            "server_key_type": SERVER_KEY_TYPE,
             "uri": key.id
         }
     ).result()
     print("Create server key:\n{}".format(server_key))
-
+    
+    # Set encryptrion protector
+    sql_client.encryption_protectors.begin_create_or_update(
+        GROUP_NAME,
+        SERVER,
+        "current",
+        {
+            "server_key_name":SERVER_KEY,
+            "server_key_type":SERVER_KEY_TYPE
+        }
+    )
```

Comment:
```suggestion
    ).result()
```
call `.result()` to make sure the operation is complete before next step.